examples/nodejs/aws/eventbridge/webapp/src/components/InfoModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { Help } from "../svgs/help";
+
+type InfoModalProps = {
+  isVisible: boolean;
+  onClose: () => void;
+};
+
+const appUsageInfo = `
+<strong>Application Overview:</strong><br>
+The application showcases a write-through cache pattern implementation for DynamoDB utilizing DynamoDB Streams, AWS EventBridge, and Momento. It enables users to perform create, update, and delete operations on items within a DynamoDB table, with real-time reflection of changes in the cache (<span class="italic">"momento-eventbridge-cache"</span>) and topic (<span class="italic">"momento-eventbridge-topic"</span>).<br><br>
+
+<strong>How to Use the App:</strong><br><br>
+<strong>Prerequisites:</strong><br>
+Before interacting with the application, ensure the existence of the cache <span class="italic">"momento-eventbridge-cache"</span>. If the cache does not exist, it can be created via <a class="text-blue-500" href="https://console.gomomento.com" target="_blank" rel="noopener noreferrer">Momento Console</a>.<br><br>
+
+<strong>Getting Started:</strong><br>
+1. <strong>Create Weather Record:</strong> Begin by creating a weather record within the DynamoDB table. The application does not enforce any constraints on the values entered, allowing users to input any desired data.<br>
```

Comment:
`by creating a weather record within the DynamoDB table` -> `using the web form to create an item in the WeatherRecords DynamoDB table that we created with our CDK stack. Once the item is saved to DynamoDB, this will trigger the EventBridge event, which will automatically write-through the value to the Momento cache, as well as publish a notification to the Momento topic.`